modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/widget/WidgetManagerImpl.kt

```diff
@@ -192,4 +214,10 @@ class WidgetManagerImpl @Inject constructor(
     private fun getSkipForwardIntent(): PendingIntent? {
         return MediaButtonReceiver.buildMediaButtonPendingIntent(context, PlaybackStateCompat.ACTION_SKIP_TO_NEXT)
     }
+
+    private fun getRemoteViewsLayoutId() = if (settings.useDynamicColorsForWidget.value) {
+        R.layout.widget_dynamic_colors_theme
+    } else {
+        R.layout.widget_default_theme
+    }
```

Comment:
Since we [cannot change the app widget theme dynamically](https://stackoverflow.com/a/4501902/193545), I switch between the layout with the correct theme. Both layouts include a common widget layout, they just have different themes applied to them.

`<include layout="@layout/widget" />`